YACS-RCOS/yacs.n
# Fixed sis_course_fetch typo

**Issue**

Fixes issue in #333 

**Test Procedure**

> 1. Compile a new Spring 2021 CSV & Ensure it compiles properly
> 2. Upload the CSV in the admin panel of your test enviroment
> 3. Check that ECSE 2050 & ECSE 2100 & ECSE 2610 display correctly
